src/utils.py

```diff
@@ -86,7 +86,7 @@ def process_tpf_query_text(resp, raw_repo_uri, call_name, extraMetadata):
         pagination_param['type'] = "int"
         pagination_param['in'] = "query"
         pagination_param['description'] = "The page number for this paginated query ({} results per page)".format(pagination)
-        params.append(pagination_param)
+        # params.append(pagination_param) # Does this get used elsewhere?
```

Comment:
I also added it to `item`
```
    item = {
...
        'params': params,
...
    }
```
because it is what we also do in `process_sparql_query_text`.